src/main/app/case/types/task-list/enum/taskStatus.ts

```diff
@@ -0,0 +1,4 @@
+export enum TaskStatus {
+  Complete = 'taskList:taskStatus.complete',
+  NotStarted = 'taskList:taskStatus.notStarted'
```

Comment:
The view is still responsible for the translation itself - 

`<strong class="govuk-tag task-list__tag" id="{{ sectionId + '-' + loop.index}}">{{ t(task.status) }}</strong>`

I think for now it works. @KiranVarma02 @mounikahmcts thoughts?

Reply:
does `taskList:taskStatus.` have to be in this enum?

Reply:
taskStatus can be of three fixed (two for now) options. It does not have to be an enum but if it's of fixed value then it's perfectly reasonable to use enum rather than type `string`.

Reply:
Sorry, I didn't mean that. It's fine to be enum for status { Complete, Not started }. does this have to be in enum too - `taskList:taskStatus.`?